CHANGELOG.md

```diff
@@ -20,6 +19,12 @@ CHANGELOG
 **CHANGES**
 - Upgrade Cinc Client to version to 18.4.12 from 18.2.7.
 
+3.9.3
+------
+
+**ENHANCEMENTS**
+- Add support for FSx Lustre as a shared storage type in us-iso-east-1.
```

Comment:
Could you also please add this: https://github.com/aws/aws-parallelcluster/issues/6237

Reply:
Already added by https://github.com/aws/aws-parallelcluster/commit/6e89b9ae6764438fff83b5b5648bc37cdace2d09